src/components/CaraouselHome/index.js

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import "./Carousel.css";
+import { SLIDE_INFO } from "./../../constant/properties";
+
+import Box from "@material-ui/core/Box";
+import Fade from "@material-ui/core/Fade";
+import Paper from "@material-ui/core/Paper";
+
+// const data = [
```

Comment:
Don't have commented code in the PR.

Reply:
yes, by mistake I missed this. will be more conscious.